README.md

```diff
@@ -17,18 +17,27 @@
 
 This project is a collection of templates for Plone integrators to use through [Cookieplone](https://github.com/plone/cookieplone "Link to the GitHub repository of Cookieplone").
 
+
 ## Prerequisites
 
-- **[pipx](https://pipx.pypa.io/stable/ "Link to the website of pipx")**: A tool for installing and running Python applications.
+-   [uv](https://docs.astral.sh/uv/) is the recommended tool for managing Python versions and project dependencies.
+-   Node.js and pnpm: essential for managing and running JavaScript packages.
+
 
-## Installation 💾
+### uv
 
-Install `pipx` with the following command:
+To install uv, use the following command, or visit the [uv installation page](https://docs.astral.sh/uv/getting-started/installation/) for alternative methods.
 
 ```shell
-pip install pipx
+curl -LsSf https://astral.sh/uv/install.sh | sh
 ```
 
+
+### Node.js
+
+Follow the [Plone documentation](https://6.docs.plone.org/install/install-from-packages.html#pre-requisites-for-installation) for detailed instructions.
```

Comment:
Node.js is a prerequisite for running some of the commands that are generated by these templates (in the context of the new repository), but it is not a prerequisite for running cookieplone.

I'm not sure if we should include it here. It's a bit weird to say you have to install Node, when you won't need it at all if you only generate a backend addon or Classic UI project.

And the templates which do require Node already have a preflight check to tell you if it's missing. If anything we should just make sure that output points to the information needed to fix the problem.

Reply:
Done in 94106c7e2b4425f780fb0a94e01c3d6a020403a7.

I haven't checked what Cookieplone says if you don't have the required version of Node.js installed, but I trust it is helpful enough to lead the user in the right direction.